src/h5web/visualizations/shared/utils.ts

```diff
@@ -45,37 +45,52 @@ export function computeVisSize(
     : { width, height: width / aspectRatio };
 }
 
+export function getExtent(
+  values: number[],
+  initialExtent?: [number, number, number]
+): [number, number, number] {
+  return values.reduce((acc, val) => {
+    const [accMin, accMax, accPositiveMin] = acc;
+    return [
+      Math.min(val, accMin),
+      Math.max(val, accMax),
+      val > 0 ? Math.min(val, accPositiveMin) : accPositiveMin,
```

Comment:
I tried to lighten a bit the code here by using `Math.min`, `Math.max` and setting initially `accPositiveMin` to `Infinity`.

Reply:
Haha nice, not sure why I didn't think of that. 🤦‍♂️